src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/src/fail_driver1.c

```diff
@@ -0,0 +1,253 @@
+/*++
+
+Copyright (c) Microsoft Corporation.  All rights reserved.
```

Comment:
Is it ok for all these files under StaticDriverVerifier/src to be in a public repo?